### PR TITLE
Upgrade spring-cloud-context to 2.0.0.RELEASE

### LINKED ISSUE #346: Outdated dependency to spring-cloud-context
Hey!

https://github.com/spring-cloud/spring-cloud-aws/blob/master/spring-cloud-aws-dependencies/pom.xml points to spring-cloud-context 1.3.2.RELEASE, which is incompatible with Spring Boot 2.0.x.

When running an application using Spring Boot 2.0.3 and spring-cloud-aws 2.0.0.RELEASE, I get:

```
15:43:43.197 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
java.lang.NoSuchMethodError: org.springframework.boot.builder.SpringApplicationBuilder.<init>([Ljava/lang/Object;)V
        at org.springframework.cloud.bootstrap.BootstrapApplicationListener.bootstrapServiceContext(BootstrapApplicationListener.java:161)                                                                                                            at org.springframework.cloud.bootstrap.BootstrapApplicationListener.onApplicationEvent(BootstrapApplicationListener.java:102)
        at org.springframework.cloud.bootstrap.BootstrapApplicationListener.onApplicationEvent(BootstrapApplicationListener.java:68)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:127)
        at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:74)
        at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:54)
        at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:358)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:317)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1255)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1243)
```

### PR DESCRIPTION
**Pull request by mziemba**
Fixes #346

### CODE REVIEW

File: spring-cloud-aws-dependencies/pom.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		<aws-java-sdk.version>1.11.336</aws-java-sdk.version>
 		<elasticache.version>1.1.1</elasticache.version>
 		<jmemcached.version>1.0.0</jmemcached.version>
-		<spring-cloud-context.version>1.3.2.RELEASE</spring-cloud-context.version>
+		<spring-cloud-context.version>2.0.0.RELEASE</spring-cloud-context.version>
```
Comment (artembilan) [inline spring-cloud-aws-dependencies/pom.xml]: That's indeed nasty bug 😢 .

I wonder why `parent` `<artifactId>spring-cloud-dependencies-parent</artifactId>` doesn't do the trick to resolve the version for the `spring-cloud-context` automatically...

Comment (mziemba) [reply spring-cloud-aws-dependencies/pom.xml]: As far as I can see there are no dependency versions defined in spring-cloud-dependencies-parent

Comment (artembilan) [reply spring-cloud-aws-dependencies/pom.xml]: OK. I see.
Now the question: why this `spring-cloud-context` here in the BOM at all?
It is not a part of this project and we don't manage here its version. It has to come to the target project via the proper `spring-cloud-dependencies` BOM.

I mean that I suggest to remove this dependency from here.

Comment (mziemba) [reply spring-cloud-aws-dependencies/pom.xml]: When I try to remove it:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.springframework.cloud:spring-cloud-context:jar is missing. @ org.springframework.cloud:spring-cloud-aws-parameter-store-config:[unknown-version], /Users/ziembam/Workspace/Code/spring-cloud-aws/spring-cloud-aws-parameter-store-config/pom.xml, line 42, column 15
```

so looks like this version is used in spring-cloud-aws-parameter-store-config

Comment (artembilan) [reply spring-cloud-aws-dependencies/pom.xml]: I see. So, that one uses correct:
```
<parent>
		<groupId>org.springframework.cloud</groupId>
		<artifactId>spring-cloud-aws</artifactId>
		<version>2.0.2.BUILD-SNAPSHOT</version>
</parent>
```
but what is missed there is something like this:
```
<dependency>
				<groupId>org.springframework.cloud</groupId>
				<artifactId>spring-cloud-commons-dependencies</artifactId>
				<version>${spring-cloud-commons.version}</version>
				<type>pom</type>
				<scope>import</scope>
</dependency>
```
Which really will for us that `spring-cloud-context`, but again: we must not provide such a dependency in our BOM.

Comment (mziemba) [reply spring-cloud-aws-dependencies/pom.xml]: I've move the spring-cloud-context dependency from spring-cloud-aws-dependencies BOM to spring-cloud-aws parent pom

**Comment by mziemba** [comment]
@artembilan I've moved the spring-cloud-context dependency from spring-cloud-aws-dependencies BOM to spring-cloud-aws parent pom

**Comment by maciejwalkowiak** [comment]
Thanks @mziemba for contribution. This issue is not present anymore when using `2.2.1` version.

**Pull request closed without merging**